src/features/pilot_management/Roster/index.vue

```diff
@@ -171,7 +171,7 @@ export default Vue.extend({
     drag: false,
     newGroupMenu: false,
     tempGroups: [],
-    shown: [],
+    hidden: [],
```

Comment:
Flipping from `shown` to `hidden` so that newly-imported pilots & their groups are immediately visible, without needing to push the pilot to "shown".